packages/inference/src/snippets/getInferenceSnippets.ts

```diff
@@ -266,6 +268,11 @@ const prepareImageToImageInput = (model: ModelDataMinimal): object => {
 	return { inputs: data.image, parameters: { prompt: data.prompt } };
 };
 
+const prepareImageToVideoInput = (model: ModelDataMinimal): object => {
+	const data = JSON.parse(getModelInputSnippet(model) as string);
```

Comment:
Could you add a simple example in https://github.com/huggingface/huggingface.js/blob/main/packages/tasks/src/snippets/inputs.ts ? Something like this:

```ts
const inputsImageToVideo = () => `{
    "image": "cat.png",
    "prompt": "Make the cat dance."
}`;
```

and then 

```ts
    "image-to-video": inputsImageToVideo,
```
in the `modelInputSnippets` mapping.

This way, `const data = JSON.parse(getModelInputSnippet(model) as string);` will return a valid payload to generate the inference snippets.